Automattic/wp-calypso
# Social Previews | Simplify Facebook preview components

Social Previews package has confusing component structure and names.
- `FacebookPreview`
- `FacebookFullPreview`
- `FacebookLinkPreview`
- `FacebookDefaultLinkPreview`
- `FacebookPostPreview`
- `FacebookDefaultPostPreview`

Making it difficult to understand what does each component do.

## Proposed Changes

This PR simplifies the names and components and only keeps these exports

- `FacebookPreview`
- `FacebookFullPreview`
- `FacebookLinkPreview`
- `FacebookPostPreview`

In another PR I will further simplify the exports for all other preview components to make the package generic and fully customizable.

## Testing Instructions

- Make sure you have a self-hosted site connected to Jetpack, with a subscription that includes Social.
- Spin up Calypso, by running this branch locally or using a live link below.
- Go to `/marketing/traffic/:site` for a self hosted site
- Scroll down to "Website Meta" section
- Click on "Show Previews" to open the previews modal
- Confirm that the Facebook preview looks as before.

## Pre-merge Checklist
